packages/dapp/src/components/Account/index.tsx

```diff
@@ -92,6 +111,17 @@ export default class AccountRouter extends React.Component<RouteComponentProps>
     window.location.reload();
   };
 
+  public handleOnAddWallet = (): void => {
+    const { history } = this.props;
+
+    history.push({
+      pathname: `/tokens`,
+      state: {},
+    });
+    // TODO(jorgelo): Same as above, there should be a better way to force a refetch.
```

Comment:
Check this again after we merge my PR https://github.com/joincivil/Civil/pull/867 with updated AccountEthAuth component - the `update` property passed to `<Mutation>` updates the cached value for the `currentUser { ethAddress }` query, which I damn hope also updates the cached value for the larger `currentUser` query you use, so it should update state without refresh.